client/src/blog/post.tsx

```diff
@@ -0,0 +1,152 @@
+import useSWR from "swr";
+
+import { HydrationData } from "../../../libs/types/hydration";
+import { HTTPError, RenderDocumentBody } from "../document";
+import { CRUD_MODE } from "../env";
+
+import "./index.scss";
+import "./post.scss";
+import {
+  BlogImage,
+  BlogPostData,
+  BlogPostFrontmatter,
+} from "../../../libs/types/blog";
+import { useCopyExamplesToClipboard } from "../document/hooks";
+
+function MaybeLink({ link, children }) {
+  return link ? (
+    link.startsWith("https://") ? (
+      <a href={link} className="external" target="_blank" rel="noreferrer">
+        {children}
+      </a>
+    ) : (
+      <a href={link}>{children}</a>
+    )
+  ) : (
+    <>{children}</>
+  );
+}
+
+export function TimeToRead({ readTime }: { readTime: number | undefined }) {
+  if (!readTime) {
+    return <></>;
+  }
+  return <span className="read-time">~{readTime} minute read</span>;
```

Comment:
request from slack: https://mozilla.slack.com/archives/G018PLBEUES/p1683095649977879?thread_ts=1683094627.019479&cid=G018PLBEUES
```suggestion
  return <span className="read-time">{readTime} minute read</span>;
```